docs/source/tutorial_task.md

```diff
@@ -399,259 +333,125 @@ The sample `setup_data` method for our task is presented below.
 ```python
 def setup_data(self, path):
     print('loading: ' + path)
-
-    # open data file with labels
-    # (path will be provided to setup_data from opt['datafile'] defined above)
-    with open(path) as labels_file:
-        self.labels = json.load(labels_file)
-
-    # define standard question, since it doesn't change for this task
-    self.question = 'Which number is in the image?'
-    # every episode consists of only one query in this task
-    new_episode = True
-
-    # define iterator over all queries
-    for i in range(len(self.labels)):
-        # set up path to curent image
-        img_path = os.path.join(self.image_path, '%05d.bmp' % i)
-        # get current label, both as a digit and as a text
-        label = [self.labels[i], self.num_strs[int(self.labels[i])]]
-        # yield tuple with information and new_episode? flag (always True)
-        yield (self.question, label, None, None, img_path), new_episode
+    with PathManager.open(path) as data_file:
+        self.squad = json.load(data_file)['data']
+    for article in self.squad:
+        # each paragraph is a context for the attached questions
+        for paragraph in article['paragraphs']:
+            # each question is an example
+            for qa in paragraph['qas']:
+                question = qa['question']
+                answers = tuple(a['text'] for a in qa['answers'])
+                context = paragraph['context']
+                yield (context + '\n' + question, answers), True
```

Comment:
can we instead switch to
```python
yield {"text": content + "\n" + question, "labels": answers}, True
```

I want to encourage this because it makes it easy for users to see how to add special fields, which was a case the CAIR team needs a lot

Reply:
sounds good